src/Parser/Monk/Brewmaster/Modules/Spells/IronSkinBrew.js

```diff
@@ -57,7 +111,20 @@ class IronSkinBrew extends Analyzer {
     }
   }
 
+  // returns true if we hit the WCL bug with 100%/0% uptime on ISB
+  get wclUptimeBug() {
```

Comment:
You could fix this perhaps more consistently with a "Normalizer". Normalizers modify the events by changing, removing or adding new fabricated events to keep it more consistent and solve bugs in the combatlog. One such instance that we always considered was buffs not appearing despite the spell casts that should trigger them was being cast. In such instances you could fabricate an `applybuff`.

In fact if the ISB cast still triggers an `refreshbuff` event, then the `ApplyBuff` Normalizer should already take care of fabricating an `applybuff` even for you. Did you verify this in a broken log?

Reply:
I hadn't thought of using a normalizer for that. I'll look into doing so.

I did confirm this with a broken log (and that my hack works around it, showing 100% ISB uptime after the log is finished parsing).